src/Core/Dependencies.js

```diff
@@ -6630,6 +6626,19 @@ class CountAmongSiblingsDependency extends Dependency {
     } else {
       this.componentName = this.upstreamComponentName;
     }
+    if (this.definition.componentType) {
+      this.componentType = this.definition.componentType;
+    } else if (this.definition.sameType) {
+      this.sameType = true;
+    }
+
+    if (this.definition.excludeComponentTypes) {
```

Comment:
I see a number of references in the new code in components referring to excludeComponentTypesForEnumeration, but it's not clear to me yet why we needed to add this code here related to the property excludeComponentTypes.

Reply:
Discussed with Duane, it looks like we might change how this works after feedback from the other professors, but the code I was not seeing in reference to my first comment was in returnDepedencies() in SectioningComponent.js